src/session.ts

```diff
@@ -191,6 +195,9 @@ export class SessionManager implements Middleware {
 
```

Comment:
right above this there's a `-HostName`... maybe we can change that too?

`"${this.HostName} VS Code Host"`?

Reply:
That gives:
```
-HostName 'PowerShell Preview Visual Studio Code Host'
```
How about this instead?
```
-HostName 'Visual Studio Code with PowerShell Preview extension'
```

Reply:
Also could abbreviate that some:
```
-HostName 'Visual Studio Code w/PowerShell Preview ext'
```

Reply:
Wait a tic, searching for that host name string, I see this in the changelog:
```
`$host.Name` now returns "Visual Studio Code Host"
```
OK, so folks key off that name to configure their profile.  Duh, I've done the same thing.  Here's code from my profile:
```powershell
if (($Host.Name -eq "ConsoleHost") -or ($Host.Name -eq "Visual Studio Code Host")) {
    ...
}
```
I don't think we want to change this string or we will break folks' profiles.

Reply:
Maybe instead, we could include the "preview" info in the $HostVersion parameter?

Reply:
Well scratch that idea:
````csharp
        // Fix the host version so that PowerShell can consume it.
        // This is needed when the extension uses a prerelease
        // version string like 0.9.1-insiders-1234.
        this.HostVersion = this.HostVersion.split("-")[0];
```

Reply:
Ugh, I really wish the .NET Core team would have sucked this one up and provided semver support from day one.  One option would be to supply a number for the fourth part (revision) only for previews so a preview would have a "distinguishable" version number that works with the System.Version class e.g. preview version would be something like `2020.4.19.1` and stable would be `2020.4.20`. I wouldn't make this change in this PR but **if** we wanted to give users a way to `detect` stable vs preview in their profile - I think this is how we would want to do that.  We do not want to change the `$host.Name` string at this point.

Reply:
Yeah let's not mess with that.